### ISSUE TITLE
Timeout passing is not overriding

### ISSUE THREAD

**Issue by rabeesh** [One-com/node-mocha-docker-postgres#5]
If current mocha timeout is greater then `10000`, would it possible to override this default value `10000` like below

```
if (this.timeout() < 10000) {
       this.timeout(10000);
}
```
Currently it's failing tests in webshop

https://github.com/One-com/node-mocha-docker-postgres/blob/60e48413c4b986f3f7b023171fd27388a71e9eb0/index.js#L79

**Comment by gustavnikolaj** [comment]
You should be able to override the timeout after calling the docker postgres hook in your before block. :-) If that doesn't work we can make it an option.

**Comment by rabeesh** [comment]
We need to override this default value `10000`. 

Currently we are passing `--timeout` as `50000` for mocha, but docker docker postgres is timed out after `10000`.

**Comment by rabeesh** [comment]
The problem we are currently facing in timeout `10000` in dockerPostgres context is getting timed out , even though we set from outside.


eg: we are running a mocha test with  `--timeout 50000`

But dockerPostgres is not overriding with that, it still be 10000 and with that timeout it is failing.

**Comment by gustavnikolaj** [comment]
Added a fix in v1.0.0 that will only set the timeout to 10000 if the current timeout is below 10000.
